plugins.js

```diff
@@ -12,33 +12,34 @@ var util        = require('util');
 var states      = require('./connection').states;
 
 var plugin_paths = [path.join(__dirname, './plugins')];
-if (process.env.HARAKA) { plugin_paths.unshift(path.join(process.env.HARAKA, 'plugins')); }
+if (process.env.HARAKA) {
+    plugin_paths.unshift(path.join(process.env.HARAKA, 'plugins'));
+}
 
 function Plugin(name) {
     this.name = name;
     this.base = {};
-    this.timeout = config.get(name + '.timeout');
-    if (this.timeout === null) {
-        this.timeout = config.get('plugin_timeout') || 30;
-    }
-    else {
-        logger.logdebug("plugin " + name + " set timeout to: " + this.timeout + "s");
+    this.timeout = parseFloat((config.get(name + '.timeout')));
+    if (!this.timeout) {
```

Comment:
Hmmm, maybe. IIRC, I put that there after ending up with a timeout that was NaN. So maybe this would be better?
```js
if (this.timeout === NaN)
```